test/models/article_test.exs

```diff
@@ -15,4 +15,9 @@ defmodule EspiDni.ArticleTest do
     changeset = Article.changeset(%Article{}, @invalid_attrs)
     refute changeset.valid?
   end
+
+  test "changeset with url missing" do
```

Comment:
Should it rather say sth.like: "changeset with url missing is invalid" to make it a bit more clear/readable?